tflite2onnx/op/resize.py

```diff
@@ -0,0 +1,128 @@
+import logging
+import tflite
+import numpy as np
+
+from tflite2onnx.layout import Layout
+from tflite2onnx import mapping
+from tflite2onnx.op.common import Operator
+
+logger = logging.getLogger('tflite2onnx')
+
+
+class Resize(Operator):
+    TypeMapping = {
+        tflite.BuiltinOperator.RESIZE_NEAREST_NEIGHBOR: 'Resize',
```

Comment:
As we are enabling `RESIZE_NEAREST_NEIGHBOR` here too, it would be great to have a dedicated TFLite model for test.